File: src/files/DuckNumber.py
def check_duck(num) :
	n = len(num)	
	i = 0
	while (i < n and num[i] == '0') :
		i = i + 1
	while (i < n) :
		if (num[i] == "0") :
			return True
		i = i + 1
	return False

File: src/files/DuckNumber.cpp
bool check_duck(string num)
{
	int i = 0, n = num.length();
	while (i < n && num[i] == '0')
		i++;
	while (i < n) {
		if (num[i] == '0')
			return true;
		i++;
	}
	return false;
}